public/app/features/explore/RichHistory/RichHistoryStarredTab.tsx

```diff
@@ -3,7 +3,7 @@ import React, { useEffect } from 'react';
 
```

Comment:
Because starring queries is not enabled yet, I am not going to review this functionality until reviewing the starring PR. Codewise though it looks good (except for the nit with the period at Showing x of y.)

Reply:
One way would be to star items in the database or run the script to generate items in local storage and re-do the migration (see #48572).